Content/LeagueSandbox-Scripts/Champions/Akali/Q.cs

```diff
@@ -26,6 +34,7 @@ public void OnDeactivate(IObjAiBase owner, ISpell spell)
 
         public void OnSpellPreCast(IObjAiBase owner, ISpell spell, IAttackableUnit target, Vector2 start, Vector2 end)
         {
+            ApiEventManager.OnSpellMissileHit.AddListener(this, new KeyValuePair<ISpell, IObjAiBase>(spell, owner), TargetExecute, true);
```

Comment:
Move this to OnActivate and change `true` to `false`. Otherwise, if you attempt to fire two of the projectiles, a single projectile will do damage worth two.